Sources/ParseSwift/Types/ParseGeoPoint.swift

```diff
@@ -75,6 +75,17 @@ public struct ParseGeoPoint: Codable, Hashable {
         self.latitude = location.coordinate.latitude
         try validate()
     }
+
+    /**
+      Creates a new `ParseGeoPoint` instance for the given `CLLocationCoordinate2D`, set to the location's coordinates.
+       - parameter location: Instance of `CLLocationCoordinate2D`, with set latitude and longitude.
+     - throws: `ParseError`.
+     */
```

Comment:
Can you change the name of “locationCoordinate” to “coordinate”?

Reply:
ok.

Reply:
did you mean CLLocationCoordinate2D to CLCoordinate2D?